README.md

```diff
@@ -119,72 +151,95 @@ calculations. Before the TLS connection is complete, a shared secret key
 must be negotiated between the server and the client. This key is used
 to encrypt all future communications during the connection.
 
-### **MQTT Connection** 
+### MQTT Connection
 
 After successfully connecting on the TLS level, the board starts
 establishing the MQTT connection. Since the TLS handles authentication
 and security, MQTT does not have to provide a username or password.
 
 <img src=".//media/image8.ppm"/>
 
-Checklist
-=========
+## Prerequisites
+
+- [Setup your Azure IoT hub](https://github.com/Azure/azure-iot-device-ecosystem/blob/master/setup_iothub.md)
+- [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/releases)
+- For the Plug and Play sample, download the Plug and Play DTMI [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json)
+- [MPLAB X IDE V5.30 or later](https://www.microchip.com/mplab/mplab-x-ide)
+- [XC16 Compiler v1.50 or later](https://www.microchip.com/mplab/compilers)
+
+## Sample Descriptions
+
```

Comment:
Could you add a brief sentence here to clarity with the audience that these are two independent samples for two different end goals. For example:

"Currently, we provides two samples with two different end goals. One shows case how the PIC-IoT board interacts with DPS. The other shows case IoT Plug and Play's benefits on the board. Depending on your end-goals, pick one sample to try out. The instruction below applies to both samples, the only different is at the "View PIC-IoT board telemetry on Azure IoT Explorer" step.

Note that the two samples a building on top of each other. Eventually, they will be merge into one".